migrations/Version202207201131371465_qtiItemPci.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\qtiItemPci\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\qtiItemPci\model\IMSPciModel;
+use oat\qtiItemPci\scripts\install\RegisterPciMathEntry;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202207201131371465_qtiItemPci extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return 'Register new minor version of Math Entry PCI with fix: math expressions in prompt should be rendered correctly';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $registry = (new IMSPciModel())->getRegistry();
```

Comment:
where do you use this `$registry` variable? it is useless

Reply:
You are right, it was also copy-pasted. Will remove it.